src/components/main-form.js

```diff
@@ -124,80 +167,195 @@ class MainForm extends React.Component {
-              name="options.o1.interestRate"
-              placeholder="APR"
-              type="number"
-              value={o1.interestRate}
-              onChange={this.updateInput}
+            <Accordion
+              title="What is an asset and types of assets"
+              body="Some quick explanation for what this is"
             />
-            <Input
-              name="options.o2.interestRate"
-              placeholder="APR"
-              type="number"
-              value={o2.interestRate}
-              onChange={this.updateInput}
+            <Accordion
+              title="Expected/Average returns by asset"
+              body="Some quick explanation for what this is"
             />
-          </Row>
-        </div>
-        <div className="section">
-          <h5>Return on Investment (ROI)</h5>
-          <Input
-            name="investmentRate"
-            placeholder="ROI"
-            type="number"
-            value={this.state.investmentRate}
-            onChange={this.updateInput}
-          />
-        </div>
-        <div className="section">
-          <h5>Inflation</h5>
-          <Input
-            name="inflation"
-            placeholder="Inflation"
-            type="number"
-            value={this.state.inflation}
-            onChange={this.updateInput}
-          />
-        </div>
-
-        <div className="result">
-          <pre>{JSON.stringify(this.state, null, "\t")}</pre>
-        </div>
-      </div>
+            <Accordion
+              title="Volatility/Risk and Diversification"
+              body="Some quick explanation for what this is"
+            />
+            <Accordion
+              title="Long term vs short term"
+              body="Some quick explanation for what this is"
+            />
+            <Accordion
+              title="Retirement accounts vs brokerage accounts"
+              body="Some quick explanation for what this is"
+            />
+          </Section>
+
+          <Section>
+            <h6>Inflation</h6>
+            <Info>Some quick explanation for what this is</Info>
+            <InputGroup>
+              <Input
+                name="inflation"
+                placeholder="Inflation"
+                type="number"
+                value={this.state.inflation}
+                onChange={this.updateInput}
+              />
+              <InputGroupAddon addonType="append">
+                <InputGroupText>%</InputGroupText>
+              </InputGroupAddon>
+            </InputGroup>
+            <Accordion
+              title="What is inflation"
+              body="Some quick explanation for what this is"
+            />
+            <Accordion
+              title="How do you determine inflation"
+              body="Some quick explanation for what this is"
+            />
+            <Accordion
+              title="How inflation affects your mortgage and investments"
+              body="Some quick explanation for what this is"
```

Comment:
May be good to mark anything we want to swap out w/ a TODO so we can easily find it later

Reply:
what exactly are you referring to? all the text/content I added?

Reply:
"Some quick explanation for what this is"

Reply:
so would you would a TODO on every accordion? or would you just put it at the very top with a general "Need to work on text/content"?

Reply:
TODOs would be good, should be easy to search for later